settings.h

```diff
@@ -521,7 +521,11 @@ typedef enum {
     // Calculated base values for driver/plugin stepper settings
     Setting_AxisExtended0        = Setting_AxisSettingsBase2,
     Setting_AxisExtended1        = Setting_AxisSettingsBase2 + AXIS_SETTINGS_INCREMENT,
+#if ENABLE_JERK_ACCELERATION
+    Setting_AxisJerk             = Setting_AxisSettingsBase2 + 2 * AXIS_SETTINGS_INCREMENT,
+#else
```

Comment:
Setting_AxisExtended2 is used by the Trinamic code...
The extended axis settings are aimed at plugin code so should not be used by the core. I want to move Setting_AxisStepperCurrent and Setting_AxisMicroSteps to the extended settings in order to free them up for the core but I have not figured out how I can do that in a backward compatible way. So I am considering adding a new block of axis settings for core use. For now you can switch to Setting_AxisExtended8 instead?

Reply:
Will do. 
I just had it on the Setting_AxisExtended2 because it lines up with the normal Acceleration settings number, but there's no harm in moving it out of the way.